Vagrantfile

```diff
@@ -1,5 +1,18 @@
+# Vagrant > 1.8.1 is required due to
+# https://github.com/mitchellh/vagrant/issues/6793
+Vagrant.require_version ">= 1.8.3"
```

Comment:
It's the rationale for why we require that minimum version on the line below. Otherwise, I'm not sure I understand your question.